src/canister/individual_user_template/src/api/cdao/mod.rs

```diff
@@ -113,11 +111,16 @@ async fn deploy_cdao_sns(
     let (registered, limit_hit) = CANISTER_DATA.with(|cdata| {
         let cdata = cdata.borrow();
         let registered = matches!(cdata.session_type, Some(SessionType::RegisteredSession));
-        (registered, cdata.cdao_canisters.len() == CDAO_TOKEN_LIMIT)
+        (
+            registered,
+            cdata.cdao_canisters.len() == MAX_LIMIT_FOR_CREATOR_DAO_SNS_TOKEN,
```

Comment:
what will happen for already existing accounts who have greater than 2 tokens. Will their tokens be auto reduced. if not, this condition will be false and they will be able to create ?

Reply:
Good catch. Added fix.